doc/source/configuration/kayobe-automation.rst

```diff
@@ -0,0 +1,126 @@
+=================
+Kayobe Automation
+=================
+
+What is Kayobe Automation
```

Comment:
Could you rename this section "Concepts", with a subheading of "Kayobe Automation"